tests/sentry/integrations/discord/test_message_builder.py

```diff
@@ -30,12 +38,13 @@ def test_metric_alert_without_incidents(self):
                 },
             )
         )
+
         assert DiscordMetricAlertMessageBuilder(self.alert_rule).build() == {
             "content": "",
             "embeds": [
                 {
                     "title": title,
-                    "description": f"<{link}|*{title}*>  \n",
+                    "description": "",
```

Comment:
What's the difference here between `test_metric_alert_without_incidents` and `test_metric_alert_with_selected_incident`? Like when/how would a metric alert fire that has no incidents?

Reply:
I based my tests off of tests/sentry/integrations/slack/test_message_builder.py. It looks like `test_metric_alert_without_incidents` is to mean a metric alert with no active incident and never triggered while `test_metric_alert_with_selected_incident`  means a metric alert with an active incident

https://github.com/getsentry/sentry/pull/34128#issuecomment-1115407930

I am not aware of the context of when a metric alert would need to fire without incidents...

Reply:
Yeah that's odd 🤔